CONTRIBUTING.md

```diff
@@ -114,6 +123,50 @@ Examples of PR titles:
  - test: Add infinity test for `MoveEffect.to`
  - refactor: Optimize the structure of the game loop
 
+
+# Maintainers
+
+These instructions are for the maintainers of Flame.
+
+## Merging a pull request
+
+When merging a pull request, make sure that the title of the merge commit has the correct
+conventional commit tag and a descriptive title. This is extra important since sometimes the title
+of the PR doesn't reflect what GitHub defaults to for the merge commit title (if the title has been
+changed during the life time of the PR for example).
+
+All the default text should be removed from the commit message and the PR description and the
+instructions from the "Migration instruction" (if the PR is breaking) should be copied into the
+commit message.
+
+
+## Breaking changes
+
+When doing breaking changes a deprecation tag should be added when possible and contain a message
+that conveys to the user what which version that the deprecated method/field will be removed in and 
+what method they should use instead to perform the task. The version specified should be at least
+two versions after the current one, such that there will be at least one stable release where the
+users get to see the deprecation warning and in the version after that (or a later version) the
+deprecated entity should be removed.
+
+
+## Creating a release
+
+There are a few things to think about when doing a release:
+ 
+ - Search through the codebase for `@Deprecated` methods/fields and remove the ones that are marked
+   for removal in the version that you are intending to release.
+ - Create a PR containing the changes for removing the deprecated entities.
+ - Run `melos version -V <package>:<version> -V <package>:<version>` for Melos to generate a
```

Comment:
Twice the `-V`? Is it meant to be current vs next? If so we we should make that more explicit in the command

Reply:
Yes, once for each package, and the version is the version for that package. Would it be more clear if I wrote it like this?
```suggestion
 - Run `melos version -V <package1>:<version> -V <package2>:<version>` for Melos to generate a
```